docs/operator-manual/notifications/troubleshooting-commands.md

```diff
@@ -1,221 +1,6 @@
-## argocd admin notifications template get
```

Comment:
This was all duplicated from the command ref, which means a high risk of drift. Linking makes sure we send people to the most up to date info. :-)